k210.svd

```diff
@@ -998,11 +998,223 @@
             <groupName>FFT</groupName>
             <baseAddress>0x42000000</baseAddress>
             <registers>
-                <!-- TODO -->
                 <register>
-                    <name>dummy</name>
-                    <description>Dummy register: this peripheral is not implemented yet</description>
+                    <name>input_fifo</name>
+                    <description>FFT input data fifo</description>
                     <addressOffset>0x00</addressOffset>
+                    <size>64</size>
+                    <fields>
+                        <field>
+                            <name>input_fifo</name>
+                            <description>Input fifo</description>
+                            <bitRange>[63:0]</bitRange>
+                        </field>
```

Comment:
I suggest removing this field because it does not provide any useful interface to the register.
Same for `output_fifo`.

Reply:
OK